cmd/spiffe-aws-assume-role/cli/cli.go

```diff
@@ -142,11 +37,7 @@ func Run(context *CliContext, args []string) (err error) {
 	return err
 }
 
-func newKong(cli *CLI) (*kong.Kong, error) {
-	return kong.New(cli, kong.NamedMapper(mappers.Iso8601DurationMapperType, mappers.Iso8601DurationMapper{}))
-}
-
-func parse(args []string) (*kong.Context, error) {
+func Parse(args []string) (*kong.Context, error) {
```

Comment:
This is a key change: by exposing the Parse method we can allow callers to inspect and modify the resulting context before proceeding with execution.